buildsys.mk.in

```diff
@@ -954,3 +953,5 @@ DELETE_FAILED = err=$$?; printf "@TERM_EL@@TERM_SETAF1@Failed to delete @TERM_BO
 
 .CURDIR ?= .
 include ${.CURDIR}/.deps
+
+$(V).SILENT:
```

Comment:
js's buildsys.mk isn't really designed for this.  It shows way too much extraneous output, not just the commands being run.  I don't think this change can be merged in its current state.